packages/core-modules/contracts/submodules/election/ElectionCredentials.sol

```diff
@@ -46,8 +39,18 @@ contract ElectionCredentials is ElectionBase {
         }
     }
 
-    function _removeCouncilMembers(address[] calldata membersToRemove) internal {
+    function _addCouncilMembers(address[] memory membersToAdd) internal {
```

Comment:
Notice how the signature changed to accept a regular array instead of a set. This is more consistent with `_removeCouncilMembers`.